Dockerfile

```diff
@@ -1,61 +1,63 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
 ARG JOBS=1
+ARG ACTION=ci
 
 ENV DEBIAN_FRONTEND=noninteractive
 ENV TZ=Europe/Moscow
 ENV CC=clang-10
 ENV CXX=clang++-10
-ENV temp_dir /tmp/ILLIXR
-ENV opt_dir /opt/ILLIXR
-ENV illixr_nproc ${JOBS}
+ENV opt_dir=/opt/ILLIXR
+ENV illixr_nproc=${JOBS}
+ENV build_type=Release
```

Comment:
`Release` is needed for CI due to storage provisioning. Some of the deps get very large with `RelWithDebInfo`.

Reply:
Fair enough. But out of curiosity, I thought only GTSAM needed to be in Release, and the other dependencies could be in RelWithDebInfo.

Reply:
Setting `Release` for GTSAM and `RelWithDebInfo` for (some) of the others did work/fit in CI. However, this solution is ad-hoc, as we need to manually configure each plugin's build type so that we get an image that is small enough to fit. I thought using `Release` for all CI for the time being would be the easiest to maintain.